global_helpers/panther_oss_helpers.py

```diff
@@ -186,12 +186,21 @@ def kv_table() -> boto3.resource:
     return _KV_TABLE
 
 
-def get_counter(key: str) -> int:
+def ttl_expired(response: dict) -> bool:
```

Comment:
if we go with this approach you'll need to update all the ProjectionExpressions to also return the TTL. I see in a few spots we are only getting back a single column